### ISSUE TITLE
Outdated UI (vs. excalidraw.com)

### ISSUE THREAD

**Issue by Acumane** [excalidraw/excalidraw-vscode#75]
Are there plans to update the UI to be consistent with the visual redesign visible on [excalidraw.com](https://excalidraw.com/)? Is this an issue on my end?

**Comment by ad1992** [comment]
Hey no this is not an issue at your end. We have blocked the next version releases of [@excalidraw/excalidraw](https://www.npmjs.com/package/@excalidraw/excalidraw) due to some breaking changes and we will be unblocking it very soon. Once thats done you will see the same UI.

**Comment by Acumane** [comment]
@ad1992 That's good to hear! Is there anything I could contribute to help resolve this?

**Comment by PMudra** [comment]
I just discovered this extension and I am very excited about it because it simplifies my workflow! Great job!

I think there is not only a UI difference but a compatibility issue caused by the different versions used by the website and the extension? For example I have created a diagram using the website and I am not able to edit some texts using the extension. Let me know if I should create another issue for this.

**Comment by pomdtr** [comment]
Thanks for the kind words !

I was not aware that the version changes introduced some compatibility issues. @ad1992 is this expected ?

**Comment by PMudra** [comment]
Example file create using the website:

![hello excalidraw](https://user-images.githubusercontent.com/84765/207089096-2c6e2900-4cd7-4260-b7ea-8646d32665ee.png)

The text is directly attached to the arrow. And this is how it looks in the vscode extension:

![image](https://user-images.githubusercontent.com/84765/207089418-e1a531e2-82ef-4001-b43d-f9f0a2e916aa.png)

I am not able to edit the text.

**Comment by pomdtr** [comment]
It makes sense, the arrow editing feature was introduced in the past week, but the npm package was not updated since one month. Nothing I can do on my end right now, we'll have to wait for the npm package update.

**Comment by krpn** [comment]
Package was released.

**Comment by pomdtr** [comment]
Yep, I did not have much time to update it lately. PR welcome !

**Comment by huguesalary** [comment]
PR is here :) https://github.com/excalidraw/excalidraw-vscode/pull/78

**Comment by pomdtr** [comment]
Thanks, I'll take a look tomorrow !

**Comment by pomdtr** [comment]
Here is an updated version of the extension. I plan to use it for a few days before merging it (it feels quite laggy to me for an unknown reason, feel free to try it and create issues if you find any bug).

https://www.dropbox.com/s/kx57v98d9pyix29/excalidraw-editor-3.3.4.vsix?dl=0

**Comment by huguesalary** [comment]
It doesn't feel laggy on my end, but maybe I'm not opening big enough files.

Do you have an excalidraw file that feels laggy to you to share?

**Comment by pomdtr** [comment]
I just opened the one in the examples folder, but it might be related with the overhead of running the extension in debug mode

**Comment by huguesalary** [comment]
It doesn't feel laggy on my end

**Comment by pomdtr** [comment]
The version 3.4.0 of the extension with the updated UI was released.
